addons/chemical/functions/fnc_hasGasMaskON.sqf

```diff
@@ -16,4 +16,10 @@
 
 params ["_target"];
 
-goggles _target in GVAR(availGasmaskList)
+if(!isNil QGVAR(availGasmaskList) && goggles _target in GVAR(availGasmaskList)) then 
```

Comment:
`!isNil QGVAR(availGasmaskList) && goggles _target in GVAR(availGasmaskList)` is already a boolean so you can leave it with it, no if wrapping needed